lib/resources/example_json/r4_examples_location.rb

```diff
@@ -0,0 +1,189 @@
+# frozen_string_literal: true
+
+module Cerner
+  module Resources
+    R4_LOCATION_BUNDLE ||= {
+      "resourceType": 'Bundle',
+      "id": '902e1b49-b9c4-4786-b90b-c9612d478f2a',
+      "type": 'searchset',
+      "link": [
+        {
+          "relation": 'self',
+          "url": 'https://fhir-open.devcerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Location?_id=21250409'
+        }
+      ],
+      "entry": [
+        {
+          "fullUrl": 'https://fhir-open.devcerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Location/21250409',
```

Comment:
Rather than copying over the JSON here, we could actually utilize the 'R4_LOCATION_ENTRY' here. Can we give that a try please.

Reply:
Updated here: 2a8f6925fc92f6a6471674fb55164a6fc0807058